api_server/src/parsed_request.rs

```diff
@@ -160,35 +160,40 @@ pub enum Error {
     SerdeJson(serde_json::Error),
 }
 
+impl std::fmt::Display for Error {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        match *self {
+            Error::Generic(_, ref desc) => write!(f, "Error: {}", desc),
```

Comment:
Looking on how we will respond for parse request errors to clients, we use a status and a message to describe what happened. I feel that `Error:` prefix it is redundant, in the idea that there is already a response status to describe what happened.

While being redundant in a self describing response, it is useful when searching for errors in logs.

Reply:
done